src/components/Differential/DifferentialSearch.jsx

```diff
@@ -916,16 +968,26 @@ class DifferentialSearch extends Component {
           <a
             target="_blank"
             rel="noopener noreferrer"
-            href={differentialStudyHrefVisible ? differentialStudyHref : '/'}
+            href={differentialStudyHrefVisible ? differentialStudyHref : null}
           >
-            <Icon
-              name="line graph"
-              size="large"
-              className="StudyHtmlIcon"
-              inverted
-              circular
-              disabled={!differentialStudyHrefVisible}
-            />
+            <Transition
```

Comment:
to animate the report linkout, I use this transition wrapper; there sometimes seems to be some funky behavior when hovering over the icon.  i've played with it and cannot figure out anything that always plays nicely, so if yall are seeing an issue, i'd say we drop the animation for another, or entirely

Reply:
if i hover, coming up from the bottom, the tooltip flashes for like a millisecond.

Reply:
yes, wherever the popup opens (e.g. bottom center), that's where the hover has trouble. I cannot find a solution, so moving the popup to the top, which I think is the least likely place the user will approach the icon.  Most often I'm moving from the left or bottom